testing/features/support/helpers/env_variables.rb

```diff
@@ -60,7 +60,7 @@ def headless?
 
     def base_url
       if environment == :local
-        "http://ca-styleguide:6006"
+        ENV.fetch("TESTING_BASE_URL", "http://ca-styleguide:6006")
```

Comment:
This change can be permanent - We never did it here. Have no probs with us just having `ENV.fetch("TESTING_BASE_URL")` as a param with a default like you've done here.

Reply:
Cool, that was my plan if we go with this approach. But wanted to avoid needing to touch any other build config in this spike.